runtime/compiler/control/rossa.cpp

```diff
@@ -1396,6 +1396,12 @@ onLoadInternal(
    memset(aotStats, 0, sizeof(TR_AOTStats));
    ((TR_JitPrivateConfig*)jitConfig->privateConfig)->aotStats = aotStats;
 
+   bool produceRSSReportDetailed = TR::Options::getCmdLineOptions()->getVerboseOption(TR_VerboseRSSReportDetailed);
+   bool produceRSSReport = produceRSSReportDetailed || TR::Options::getCmdLineOptions()->getVerboseOption(TR_VerboseRSSReport);
+
+   if (produceRSSReport)
+      new (PERSISTENT_NEW) OMR::RSSReport(produceRSSReportDetailed);
```

Comment:
The output of this is not stored anywhere. I am guessing the constructor does some special magic, but this is rather unorthodox.

Reply:
I think it's relatively common for singleton classes...